### PR TITLE
Script to install Android SDKs

### LINKED ISSUE #30: Write logic that pre-installs Android SDKs
### 💡 Feature/Idea

Add script in `home/.chezmoiscripts/universal` that detects the presence of the Android SDK tools and then installs a configurable list of SDKs that are listed in `home/.chezmoidata.yaml`

### :thumbsup: Can you contribute?

_No response_

### PR DESCRIPTION
**Pull request by enggnr**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Provides a script to install a list of Android SDKs

- **What is the current behavior?** (You can also link to an open issue here)
No such script exists at this time

- **Other information**:
Fixes #30.

### CODE REVIEW

File: home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{{ includeTemplate "universal/profile" }}
+{{ includeTemplate "universal/logg" }}
+
+### Find the location of `sdkmanager`
+SDKMANAGER=""
+if command -v sdkmanager > /dev/null; then
+    SDKMANAGER="sdkmanager"
+elif [ -f $HOME/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager ]; then
+    SDKMANAGER="$HOME/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager"
+elif [ -f $HOME/Android/Sdk/cmdline-tools/latest/bin/sdkmanager ]; then
```
Comment (ProfessorManhattan) [inline home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl:18]: Anytime you have a variable in a string, it should be quoted... I would quote it like this "$HOME/Android/Sdk/...". However, most people in the industry quote it like this "$HOME"/Android/Sdk --- use my method to keep it consistent please.

File: home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    SDKMANAGER="$HOME/Android/Sdk/cmdline-tools/latest/bin/sdkmanager"
+else
+    logg info '`sdkmanager` is not installed or it is not available in PATH'
+fi
+
+### Run `sdkmanager`, if it is installed, to install SDKs and Tools
+if [ ${#SDKMANAGER} -gt 0 ]; then echo 'yes';fi
```
Comment (ProfessorManhattan) [inline home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl:25]: Looks like this is exiting early. Fix the if then fi fi

File: home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+### Find the location of `sdkmanager`
+SDKMANAGER=""
+if command -v sdkmanager > /dev/null; then
+    SDKMANAGER="sdkmanager"
+elif [ -f $HOME/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager ]; then
+    SDKMANAGER="$HOME/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager"
+elif [ -f $HOME/Android/Sdk/cmdline-tools/latest/bin/sdkmanager ]; then
+    SDKMANAGER="$HOME/Android/Sdk/cmdline-tools/latest/bin/sdkmanager"
+else
+    logg info '`sdkmanager` is not installed or it is not available in PATH'
```
Comment (ProfessorManhattan) [inline home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl:21]: Is there any other place it might install to on Linux?

**Review by ProfessorManhattan — COMMENTED** [review]
Couple requests on this one

File: home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl (hunk 1 above)
Comment (enggnr) [reply home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl:18]: Fixed.

File: home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl (hunk 3 above)
Comment (enggnr) [reply home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl:21]: When `cmdline-tools` are installed standalone, they can extracted to any directory. There is no installer, so there is no standard location. However, the default SDK location is `$HOME/Android/Sdk` ( `$HOME/Library/Android/sdk in MacOS`). This is the reason the script only checks in the default SDK path. This covers the case where the `cmdline-tools` are installed as part of Android Studio (when it is configured to use the default SDK location for the platform).

File: home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl (hunk 2 above)
Comment (enggnr) [reply home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl]: I left some debug code in by mistake. Fixed.

**Review by enggnr — COMMENTED** [review]
Changes complete.

**Pull request merged**